syncserver/src/middleware/sentry.rs

```diff
@@ -76,9 +72,9 @@ pub fn report_error(
             Some(e) => {
                 let metrics = MetricsWrapper::extract(sresp.request()).await.unwrap().0;
 
-                if let Some(apie) = e.as_error::<ApiError>() {
+                if let Some(apie) = e.as_error::<syncstorage_web::ApiError>() {
```

Comment:
We may want to revisit this approach? At the very least, we should double check that this is working as expected on stage before we deploy to prod